tests/TestHelpers/EmailHelper.php

```diff
@@ -26,238 +26,53 @@
-						\quoted_printable_decode($item->Content->Body)
-					);
-					return $body;
-				}
-			}
-			\usleep(STANDARD_SLEEP_TIME_MICROSEC * 50);
-			$currentTime = \time();
-		}
-
-		throw new Exception("Could not find the email to the address: " . $emailAddress);
-	}
-
-	/**
-	 *
-	 * @param string|null $localMailhogUrl
-	 * @param string|null $emailAddress
-	 * @param string|null $xRequestId
-	 * @param int|null $waitTimeSec Time to wait for the email
-	 *
-	 * @return boolean
-	 */
-	public static function emailReceived(
-		?string $localMailhogUrl,
-		?string $emailAddress,
-		?string $xRequestId,
-		?int $waitTimeSec = EMAIL_WAIT_TIMEOUT_SEC
-	):bool {
-		try {
-			self::getBodyOfLastEmail(
-				$localMailhogUrl,
-				$emailAddress,
-				$xRequestId,
-				$waitTimeSec
-			);
-		} catch (Exception $err) {
-			return false;
-		}
-
-		return true;
-	}
-
-	/**
-	 *
-	 * @param string|null $localMailhogUrl
-	 * @param string|null $emailAddress
-	 * @param string|null $xRequestId
-	 * @param int|null $emailNumber which number of multiple emails to read (first email is 1)
-	 * @param int|null $waitTimeSec Time to wait for the email
-	 *
-	 * @return mixed
-	 * @throws GuzzleException
-	 * @throws Exception
-	 */
-	public static function getSenderOfEmail(
-		?string $localMailhogUrl,
-		?string $emailAddress,
-		?string $xRequestId = '',
-		?int $emailNumber = 1,
-		?int $waitTimeSec = EMAIL_WAIT_TIMEOUT_SEC
-	) {
-		$currentTime = \time();
-		$endTime = $currentTime + $waitTimeSec;
-
-		while ($currentTime <= $endTime) {
-			$skip = 1;
-			foreach (self::getEmails($localMailhogUrl, $xRequestId)->items as $item) {
-				$thisEmailAddress
-					= $item->To[0]->Mailbox . "@" . $item->To[0]->Domain;
-				if ($thisEmailAddress === $emailAddress) {
-					if ($skip < $emailNumber) {
-						$skip++;
-						continue;
-					}
-					return $item->From->Mailbox . '@' . $item->From->Domain;
-				}
-			}
-			\usleep(STANDARD_SLEEP_TIME_MICROSEC * 50);
-			$currentTime = \time();
-		}
-		throw new Exception("Could not find the email to the address: " . $emailAddress);
-	}
-
-	/**
-	 *
-	 * @param string|null $localMailhogUrl
-	 * @param string|null $xRequestId
-	 *
-	 * @return ResponseInterface
-	 * @throws GuzzleException
-	 */
-	public static function deleteAllMessages(
-		?string $localMailhogUrl,
-		?string $xRequestId
-	):ResponseInterface {
-		return HttpRequestHelper::delete(
-			$localMailhogUrl . "/api/v1/messages",
-			$xRequestId
-		);
-	}
 
 	/**
-	 * Returns the host name or address of the Mailhog server as seen from the
+	 * Returns the host name or address of the Email server as seen from the
 	 * point of view of the system-under-test.
 	 *
 	 * @return string
 	 */
-	public static function getMailhogHost():string {
-		$mailhogHost = \getenv('MAILHOG_HOST');
-		if ($mailhogHost === false) {
-			$mailhogHost = "127.0.0.1";
+	public static function getEmailHost():string {
+		$emailHost = \getenv('EMAIL_HOST');
+		if ($emailHost === false) {
```

Comment:
I think env vars will be treated as string
```suggestion
		$emailHost = \getenv('EMAIL_HOST');
		if (($emailHost === '' || $emailHost === 'false') {
```

Reply:
func `getenv` return boolean(`false`) if the env is not set